client/wfnews-war/src/main/angular/src/assets/data/appConfig.json

```diff
@@ -67,6 +67,7 @@
         "AGOLactiveFirest": "https://services6.arcgis.com/ubm4tcTYICKBpist/ArcGIS/rest/services/BCWS_ActiveFires_PublicView/FeatureServer/0/",
         "evacDefaultUrl": "https://www.emergencyinfobc.gov.bc.ca/current-wildfire-information-2023",
         "contactInformation": {
+            "rofPhoneNumber": "1 800 663-5555",
```

Comment:
How is this different then the number directly below it? Aside from the obvious, why do we have this in there twice? Might make more sense to split the phone number and cell * number

Reply:
I believe the below one is for display in "Contact-Us" dialog desktop view only.

Reply:
Yeah, just wondering if they should be showing the same thing